tests/unit/common/test_linkedtables.py

```diff
@@ -445,7 +445,7 @@ def test_to_hierarchical_dataframe_indexed_dtr_on_last_level(self):
         p1 = DynamicTable(name='parent_table', description='parent_table',
                           columns=[VectorData(name='p1', description='p1', data=np.arange(3)), dtr_p1, vi_dtr_p1])
         # Super-parent table
-        dtr_sp = DynamicTableRegion(name='sl1', description='sl1', data=np.arange(4), table=p1)
+        dtr_sp = DynamicTableRegion(name='sl1', description='sl1', data=np.arange(3), table=p1)
```

Comment:
The previous data range was invalid because the target table only has 3 rows. This was causing an IndexError in `VectorIndex.get`.

Reply:
@oruebel since you set up this test originally, please confirm that I am not missing something